Sources/SimplenoteSearch/Query/SearchQuery.swift

```diff
@@ -60,14 +60,25 @@ public final class SearchQuery: NSObject {
         let keywords = searchText.trimmingCharacters(in: .whitespacesAndNewlines).components(separatedBy: .whitespaces)
 
         for keyword in keywords where keyword.isEmpty == false {
-            guard let tag = keyword.lowercased().suffix(afterPrefix: String.searchOperatorForTags.lowercased()) else {
+            guard let tag = checkForTagOrLocalizedTag(with: keyword) else {
                 items.append(.keyword(keyword))
                 continue
             }
-
+            
             items.append(.tag(tag))
         }
     }
+    
+    private func checkForTagOrLocalizedTag(with keyword: String) -> String? {
```

Comment:
How about...

```
private func extractTagKeyword(from keyword: String) -> String?
```

(Or something like that?). The word `Check` would imply a boolean response. And the Localized / Unlocalized could be an internal detail, just to make the api look shorter?